examples/12-fluids/00-fluids_model.py

```diff
@@ -0,0 +1,185 @@
+"""
+.. _ref_fluids_model:
+
+Explore Fluids models
+------------------------------------------------------
+
+This example shows how to explore Ansys Fluent and Ansys CFX models employing
```

Comment:
It should either be "This example shows you how to..." or "This example demonstrates..."

Reply:
@JennaPaikowsky Thanks for all the comments! I'm preparing another PR to add the fixes